framework/src/actions/CreateExecutionerAction.C

```diff
@@ -110,6 +110,7 @@ CreateExecutionerAction::act()
   Moose::setup_perf_log.pop("Create Executioner","Setup");
 
   _awh.executioner() = executioner;
```

Comment:
That would be great, there may be a few `Actions` using it but they could just get it from MooseApp instead.